src/intelli/node/stringselection.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * GTlab IntelliGraph
+ *
+ *  SPDX-License-Identifier: BSD-3-Clause
+ *  SPDX-FileCopyrightText: 2024 German Aerospace Center
+ *
+ *  Author: Jens Schmeink <jens.schmeink@dlr.de>
+ */
+#include "intelli/node/stringselection.h"
+
+#include "intelli/data/string.h"
+#include "intelli/data/stringlist.h"
+
+#include <QComboBox>
+
+using namespace intelli;
+
+StringSelectionNode::StringSelectionNode() :
+    Node(tr("String Selection")),
+    m_selection("selectedString", "selection", "selection")
+{
+    m_in = addInPort({typeId<StringListData>(), tr("List")});
+    m_out = addOutPort(makePort(typeId<StringData>()).setCaption(tr("Selection")));
```

Comment:
I like to use lower case for ports :)
```suggestion
    m_in = addInPort({typeId<StringListData>(), tr("list")});
    m_out = addOutPort(makePort(typeId<StringData>()).setCaption(tr("selection")));
```